### PR TITLE
adding more info and make the doc more readable and usable

### PR DESCRIPTION
**Pull request by navendu-pottekkat**
This is a draft PR and I will be adding changes to this. This is to easily track changes and suggest improvements.

Here are the changes that are **currently** committed to this PR:

- Add header image
- Align Shields badges to center

Merging this PR will fix #76

### CODE REVIEW

**Comment by navendu-pottekkat** [comment]
Update: Changed the first para to make it more readable and making sure that the idea of the project is conveyed.

**Comment by riibeirogabriel** [comment]
@navendu-pottekkat This png image  is so, so awesome, it will be good!

**Comment by navendu-pottekkat** [comment]
List of changes in the PR:

- Add header image
- Align badges to center
- Added install instructions
- Added usage instructions
- Improved readability

**Comment by navendu-pottekkat** [comment]
@riibeirogabriel Please review the changes

**Comment by riibeirogabriel** [comment]
@navendu-pottekkat a make some modifications, please take a look and give a feedback.

**Comment by riibeirogabriel** [comment]
Now, the readme is much better, thanks!

File: readme.md [hunk 1]
```diff
@@ -17,9 +17,10 @@
 
 With learningOrchestra, you can:
 
-* use the Database API to load dataset(CSV), convert it to JSON and store in MongoDB.
-* build prediction models with different classifiers simultaneously with Model builder API.
+* Load a dataset from an URL (in CSV format).
+* Accomplish several pre-processing tasks with this dataset.
```
Comment (navendu-pottekkat) [inline readme.md:21]: accomplish "a" is in lowercase

File: readme.md [hunk 2]
```diff
@@ -17,9 +17,10 @@
 
 With learningOrchestra, you can:
 
-* use the Database API to load dataset(CSV), convert it to JSON and store in MongoDB.
-* build prediction models with different classifiers simultaneously with Model builder API.
+* Load a dataset from an URL (in CSV format).
```
Comment (navendu-pottekkat) [inline readme.md:20]: L must be lowercase

File: readme.md [hunk 3]
```diff
@@ -36,7 +37,7 @@ And so much more! Check the [usage section](#usage) for more.
 
 *If in case, you have firewalls or other traffic-blockers, add learningOrchestra as an exception.*
 
-Ex: In Google Cloud Platform each of the VMs must allow both http and https traffic.
+Ex: In Google Cloud Platform each of the VMs must allow both `allow_http` and `allow_https` traffic rules.
```
Comment (navendu-pottekkat) [inline readme.md:40]: Ex: In Google Cloud Platform each of the VMs must allow both http and https traffic.
I think `allow_https` is understood from the above sentence and it sounds better.

**Review by navendu-pottekkat — COMMENTED** [review]
I have some suggestions, maybe it is because my OCD for documentation

File: readme.md (hunk 3 above)
Comment (riibeirogabriel) [reply readme.md:40]: @navendu-pottekkat I agree with your suggestions, please make the changes, sorry my bad english.

**Comment by navendu-pottekkat** [comment]
@riibeirogabriel Done! You can merge now

**Pull request merged**